sdk-python/tests/copilotkit/langgraph_agent/agent.py

```
@@ -0,0 +1,146 @@
+    
+    await copilotkit_exit(config)
+    return Command(
+        goto="interrupted_node",
+        update={
+            "messages": messages,
+        }
+    )
+
+
+async def interrupted_node(state: Dict[str, Any], config: RunnableConfig):
+    messages = state["messages"] + [HumanMessage(content=HUMAN_FEEDBACK_AFTER_INTERRUPT)]
```

Ruff (0.11.9)

Undefined name `HumanMessage`

(F821)